app/views/layouts/_navbar.html.erb

```diff
@@ -18,5 +18,18 @@
         <%= link_to 'Contact', contact_path, class: 'nav-link' %>
       </li>
     </ul>
+    <ul class="navbar-nav">
+      <li class="nav-item">
+      <% if user_signed_in? %>
+      <span class="mr-2">
+        <%= "Logged in as #{current_user.first_name}" %>
+      </span>
+        <%= link_to 'Log out', destroy_user_session_path, method: :delete, class: 'btn btn-primary' %>
+      <% else %>
+        <%= link_to 'Log in', new_user_session_path, class: 'btn btn-primary' %>
+        <%= link_to 'Sign up', new_user_registration_path, class: 'btn btn-primary' %>
+      <% end %>
+      </li>
```

Comment:
This looks great! Cant wait to see it in action!

Reply:
:slightly_smiling_face: